src/legacy/ui/public/saved_objects/components/saved_object_save_modal.tsx

```diff
@@ -83,38 +175,40 @@ export class SavedObjectSaveModal extends React.Component {
     });
   };
 
-  onTitleChange = (event) => {
+  private onTitleChange = (event: React.FormEvent<HTMLInputElement>) => {
```

Comment:
`React.ChangeEvent<HTMLInputElement>` (as discussed offline), same for the other events.

Reply:
done!